packages/open-next/src/core/routing/middleware.ts

```diff
@@ -53,8 +59,7 @@ export async function handleMiddleware(
   const url = initialUrl.toString();
   // console.log("url", url, normalizedPath);
 
-  // @ts-expect-error - This is bundled
-  const middleware = await import("./middleware.mjs");
+  const middleware = await middlewareLoader("./middleware.mjs");
```

Comment:
You cannot do that, when the middleware is external we use an esbuild plugin to change the resolve.
This will break it